speakers-cfp_runbook.md

```diff
@@ -1,53 +1,76 @@
+* Vote and read all the CFPs that are proposed
+* Attempt to be at every meeting for the smaller and larger team (things can change quickly and CFPs can be affected)
+* Recruit or identify possible keynote speakers
+* Work closely with the Sponsorship and Core team to make sure that the ratio of talks to workshops to open spaces is reasonable
+* Comment or filter out "sales pitches" actively through the Open CFP process
+* Administer the CFP software for DoD ATX
+* Timely answer any and all questions for that comes in from possible or future speakers
+* Attend and Run the Office Hours, pre and post CFP for new and experienced speakers
+* Finalize the speaker list
+* Send out acceptances
+* Schedule the speakers in coordination with core
+* MC the stages day of in coordination with core
 
 ## Important Dates
 
-What dates are important for this role? The following headings and details are examples.
-
 ### In the beginning
 
-* Come up with good stage jokes.
-
-### 3 Months Out
+* Copy the previous year's CFP and make any changes for the new theme from Core
+* Verify that the CFP software is ready from Global Core
+* Start discussing with the Venue team to verify options if the venue has changed for workshops and breakout spaces
 
-* Find vendors for your stage costume.
+### 7-5 Months Out
 
-### 2 Months Out
+* Prepare the CFP
+* Announce the CFP where you can. Coordinate with Core and Marketing.
+* Start discussing with possible people for Key Note slots or conversations they believe they would like to share.
+* Recruit speakers!
+* Recruit speakers!
+* Recruit speakers!
 
-* Try on the costume.
+### 4 months Out
 
-### 1 Month Out
-
-* Sell out the sponsorships.
+* Start a cadence of the Office Hours (once a week or some reasonable process)
 
-### The Week Before
+### 3-2 months Out
 
-* Panic.
+* Close down the CFP
+* Start to vote and recruit others from either outside or inside the Org team to help vote on the speakers
+* Send out the speaker acceptances
+* Continue a cadence of the Office Hours (once a week or some reasonable process)
 
-### The Day Before
+### 1 Month Out
 
-* Set up.
-* Party.
+* Verify the speakers and nothing has changed last minute
+* Continue a cadence of the Office Hours (once a week or some reasonable process)
 
 ### The Day Of
 
-* Get on stage.
+* Be prepared to MC the breakout rooms
+* Be there for the speakers and what they may need
+* Remember you are the representive of DoD ATX to many of these speakers, be there and friendly!
 
 ## Reminders or Common Mistakes
 
-What do you always try to remember when you come back to this every year? Do you often find yourself scrambling for vendors? Are you always trying to remember where you left your towel? Leave notes for the next person so we can improve :)
-
+* Spend "real time" with the CFP software before opening the CFP. It can be tricky
+* You DON'T have to "promote" someone who submitted via the CFP to a keynote slot. Recruit your keynote speaker(s) as needed.
```

Comment:
really keynotes should be a core responsibility, emphasis on feedback rather than selection/recruitment here?